plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -640,9 +643,14 @@ protected JdbcOutputTableHandle createTable(ConnectorSession session, ConnectorT
         }
     }
 
+    protected boolean setTableCommentAfterCreateTable()
```

Comment:
Can we remove this method? Or in other words, what is the reason you need it?

Reply:
Added the comment for this method.
If we use `setTableComment` we need to know when to do this. Some connectors support creating table with comment but also support  setting comment independently. Without this method we might setting the comment twice:
First set is creating table , second set is set in `setTableComment`.